src/CommunityToolkit.Maui/Extensions/HttpClientExtensions.shared.cs

```diff
@@ -21,7 +21,7 @@ public static async Task<Stream> DownloadStreamAsync(this HttpClient client, Uri
 		}
 		catch (Exception ex)
 		{
-			Trace.WriteLine($"Error getting stream for {uri}: {ex}");
+			Trace.TraceError($"Error getting stream for {uri}: {ex}");
```

Comment:
Please do not use `TraceError` in the library. It causes crash reporting tools like sentry.io to report it as an unhandled exception. Leading to our library causing erroneous bug reports for our users.